package.json

```diff
@@ -43,6 +44,8 @@
   },
   "scripts": {
     "build": "babel src --out-dir .",
+    "dev": "babel src --out-dir ../mattermost-webapp/node_modules/mattermost-redux",
+    "dev:watch": "babel --watch src --out-dir ../mattermost-webapp/node_modules/mattermost-redux",
```

Comment:
Is it possible to have the location of the webapp repo configurable via an environment variable?